docs/fundamentals/systems/form/validate.md

```diff
@@ -130,6 +130,33 @@ class EqualsText extends Validator {
 }
 ```
 
+The `execute()` method returns a boolean by default, but it can also be customized to return enums instead. In this example below, we customize the method to return one among some possible enums as strings.
+
+```js
+class EnumOutComeValidator extends Validator {
+  static validatorName = 'EnumOutCome';
+
+  execute(modelValue, foodCalories) {
+    return new Promise(resolve => {
```

Comment:
this is a bit overhead and complex for the example, but it would be ok if you rmv this execute method and rename `mealSize ` below to `execute` 👍

Reply:
Then (see in the tests), we should also show how to consume the enum outcome in `static async getMessage`